services/cypress/support/accessibilityCommands.ts

```diff
@@ -0,0 +1,32 @@
+function terminalLog(violations: Record<string, any>[]) {
+    console.log(violations)
+    cy.task(
+        'log',
+        `${violations.length} accessibility violation${
+            violations.length === 1 ? '' : 's'
+        } ${violations.length === 1 ? 'was' : 'were'} detected`
+    )
+    // pluck specific keys to keep the table readable
+    const violationData: Record<string, any>[] = violations.map(
+        ({ id, impact, description, nodes }) => ({
+            id,
+            impact,
+            description,
+            nodes: nodes.length
+        })
+    )
+
+    cy.task('table', violationData)
+}
+
+Cypress.Commands.add(
+    'checkA11yWithWcag22aa',
+    () => {
+        cy.checkA11y('', {
+            runOnly: {
+                type: 'tag',
+                values: ['wcag22aa']
```

Comment:
I don't think this is working - I added basic accessibility violations (like an image missing alt text, a button with no text content) and there were no errors. When I change this value I start to get errors again. I think it is not working for some version saw a [github issue](https://github.com/component-driven/cypress-axe/issues/123) too

Reply:
I think I'm seeing `best-practice` work lets just use that if you agree that 22aa is broken and so is wcagaa right now.

Reply:
Just to make visible our conversation. Rules for each standard are not cumulative. So in order to test all rules, all the standards must be included. If we only include `'wcag22aa'` then only the specific rules added in for that standard will be tested.

The configuration has been updated to include the complete rules 

```
      cy.checkA11y('', {
          runOnly: {
              type: 'tag',
              values: ['wcag2a','wcag2aa', 'wcag21a', 'wcag21aa','wcag22aa']
```

Reply:
Hi @JasonLin0991 and @haworku , Even in this above eg:
cy.checkA11y('', {
    runOnly: {
        type: 'tag',
        values: ['wcag2a','wcag2aa', 'wcag21a', 'wcag21aa','wcag22aa']
        

No violations are getting shown for wcag21a and wcag22aa.

Reply:
@sethims I followed the `axe-core` docs under [axe.run()](https://www.deque.com/axe/core-documentation/api-documentation/#parameters-axerun) to figure out that the tags need to be combined. You can look through the rule of each standard using [axe.getRules()](https://www.deque.com/axe/core-documentation/api-documentation/#api-name-axegetrules). Compare the rules in there with the specific rule violations you are testing for. Good luck!